project/src/main/java/com/google/sps/servlets/SellServlet.java

```diff
@@ -0,0 +1,144 @@
+package com.google.sps.servlets;
+
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Date;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.sql.DataSource;
+
+/**
+ * Receive data when an investment is sold and updated the investments and participants database.
+ */
+@WebServlet("/sell")
+public class SellServlet extends HttpServlet {
+
+  private static final Logger LOGGER = Logger.getLogger(SellServlet.class.getName());
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    long investmentID = -1;    //long investmentID = Long.parseLong(request.getParameter("id"));
+
+    DataSource pool = (DataSource) request.getServletContext().getAttribute("db-connection-pool");
+    Connection conn = null;
```

Comment:
ONLY IF YOU HAVE TIME 
from the same tutorial - https://www.mysqltutorial.org/mysql-jdbc-transaction/

look for try(Connection conn = DriverManager.getConnection(dbURL,dbUser,dbPassword);){

so it will be  try(Connection conn =pool.getConnection())

Its called The try-with-resources Statement https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html

See exmaple in this file - try (PreparedStatement statement = conn.prepareStatement(findUserIdStmt)) {